Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -362,6 +368,15 @@ void ImageEditor::keydown_event(GUI::KeyEvent& event)
 
 void ImageEditor::keyup_event(GUI::KeyEvent& event)
 {
+    if (event.key() == Key_Alt) {
+        if (!m_old_tool) {
+            set_override_cursor(Gfx::StandardCursor::None);
+            return;
+        }
+
+        set_active_tool(m_old_tool);
```

Comment:
If I just keep pressing ALT won't this leak a `PikerTool` every time?

Reply:
Yes. Resolved in latest - initialize an instance of PickerTool as part of ImageEditor, and store that as the "hotkey" instance to point to.